src/backend/joanie/settings.py

```diff
@@ -104,13 +104,16 @@ class Base(Configuration):
     # https://docs.djangoproject.com/en/3.1/topics/i18n/
 
     # Languages
-    LANGUAGE_CODE = "en-us"
+    LANGUAGE_CODE = values.Value("en-us", environ_name="LANGUAGE_CODE")
 
     # Careful! Languages should be ordered by priority, as this tuple is used to get
     # fallback/default languages throughout the app.
-    LANGUAGES = (
-        ("en-us", _("English")),
-        ("fr-fr", _("French")),
+    LANGUAGES = values.SingleNestedTupleValue(
+        (
+            ("en-us", _("English")),
+            ("fr-fr", _("French")),
+        ),
+        environ_name="LANGUAGES",
     )
```

Comment:
do we need to set this one, the language code should be enough no?

Reply:
Nope. e.g: if we want to use german as default language we should be able to update this setting to declare german.

Furthermore, language order is important for django parler. In our case, we want to set french as default language, we also have to update `LANGUAGES` to put french before english.